src/components/settings/PushNotifications/GlobalPushNotifications.tsx

```diff
@@ -37,6 +41,16 @@ import css from './styles.module.css'
 
 // UI logic
 
+const _filterUndeployedSafes = (safes: NotifiableSafes, undeployedSafes: UndeployedSafesState) => {
```

Comment:
safes here is a `{[chain]: [address1, address2]}` but since the variable name is `safes` I thought it was a safe object and later when we do difference(safes, undeployedAddresses) I was astonished that it works :)
Maybe at least in mapValues we could call it safeAddresses?